code/game/objects/items/storage/bags.dm

```diff
@@ -209,6 +209,31 @@
 		))
 ////////
 
+// -----------------------------
+//          Mini Plant Bag
+// -----------------------------
+
+/obj/item/storage/bag/plants/small
+	name = "small plant bag"
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "plantbag"
+	worn_icon_state = "plantbag"
+	resistance_flags = FLAMMABLE
+
+/obj/item/storage/bag/plants/small/Initialize(mapload)
+	. = ..()
+	atom_storage.max_specific_storage = WEIGHT_CLASS_SMALL
+	slot_flags = ITEM_SLOT_BELT | ITEM_SLOT_POCKETS
```

Comment:
why are you setting slot flags on initialize, its a variable you can just set in the definition